community/io/src/test/java/org/neo4j/io/pagecache/PageCacheTest.java

```diff
@@ -2683,251 +2808,249 @@ private void verifyReadOffsets( PageCursor cursor )
 
-        fork( closePageFile( pagedFile ) ).join();
+            fork( closePageFile( pagedFile ) ).join();
 
-        try
-        {
-            cursor.next();
-            fail( "Advancing the cursor should have thrown" );
-        }
-        catch ( FileIsNotMappedException e )
-        {
-            // Yay!
-        }
+            assertThrows( FileIsNotMappedException.class, cursor::next );
+        } );
     }
 
-    @Test( timeout = SHORT_TIMEOUT_MILLIS )
-    public void readingAndRetryingOnPageWithOptimisticReadLockingAfterUnmappingMustNotThrow() throws Exception
+    @Test
+    void readingAndRetryingOnPageWithOptimisticReadLockingAfterUnmappingMustNotThrow()
     {
-        configureStandardPageCache();
+        assertTimeout( ofMillis( SHORT_TIMEOUT_MILLIS ), () ->
+        {
+            configureStandardPageCache();
 
-        generateFileWithRecords( file( "a" ), recordsPerFilePage * 2, recordSize );
+            generateFileWithRecords( file( "a" ), recordsPerFilePage * 2, recordSize );
 
-        PagedFile pagedFile = pageCache.map( file( "a" ), filePageSize );
-        PageCursor cursor = pagedFile.io( 0, PF_SHARED_READ_LOCK );
-        assertTrue( cursor.next() );    // fault
-        assertTrue( cursor.next() );    // fault + unpin page 0
-        assertTrue( cursor.next( 0 ) ); // potentially optimistic read lock page 0
+            PagedFile pagedFile = pageCache.map( file( "a" ), filePageSize );
+            PageCursor cursor = pagedFile.io( 0, PF_SHARED_READ_LOCK );
+            assertTrue( cursor.next() );    // fault
+            assertTrue( cursor.next() );    // fault + unpin page 0
+            assertTrue( cursor.next( 0 ) ); // potentially optimistic read lock page 0
 
-        fork( closePageFile( pagedFile ) ).join();
-        pageCache.close();
-        pageCache = null;
+            fork( closePageFile( pagedFile ) ).join();
+            pageCache.close();
+            pageCache = null;
 
-        cursor.getByte();
-        cursor.shouldRetry();
-        try
-        {
-            cursor.next();
-            fail( "Advancing the cursor should have thrown" );
-        }
-        catch ( FileIsNotMappedException e )
-        {
-            // Yay!
-        }
+            cursor.getByte();
+            cursor.shouldRetry();
+            assertThrows( FileIsNotMappedException.class, cursor::next );
+        } );
     }
 
     @Test
-    public void shouldRetryFromUnboundReadCursorMustNotThrow() throws Exception
+    void shouldRetryFromUnboundReadCursorMustNotThrow() throws Exception
     {
         File file = file( "a" );
         generateFileWithRecords( file, recordsPerFilePage, recordSize );
         configureStandardPageCache();
         try ( PagedFile pf = pageCache.map( file, filePageSize );
-              PageCursor cursor = pf.io( 0, PF_SHARED_READ_LOCK ) )
+                PageCursor cursor = pf.io( 0, PF_SHARED_READ_LOCK ) )
```

Comment:
I think it looks better when these are aligned. I don't think our checkstyle has an opinion on it, though.

Reply:
those alignment shifts and tricks performed by our code style settings for idea, for sure i did not move those guys myself :nerd_face: :man_technologist: